.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+## Done
+
+[List of work items including drive-bys]
+
+## QA
+
+- Check out this feature branch
+- Run the site using the command `./run serve`
+- View the site locally in your web browser at: http://0.0.0.0:port/ (replace port with the correct port in your [project](https://canonical-web-and-design.github.io/practices/project-structure/ports.html))
```

Comment:
I was thinking something similar to http://0.0..0.0:{port}/ which should be similar to the port in the (`./env` file)[https://github.com/canonical-web-and-design/website-boilerplate/blob/master/.env].

Reply:
Makes sense. Updated